### ISSUE TITLE
Clear selection after click

### ISSUE THREAD

**Issue by hernanrz** [nmarcel/ThinkComposer#8]
When you select a concept and click to drop it, it remains "selected' so any further clicks will keep adding more, I find this very annoying as it's not normal behavior for this kind of tool, would you be willing to accept a PR to fix this?

**Comment by The-Maize** [comment]
I don't think this will get fixed the latest version of this software is full of bugs causing lot of different errors. Every one here is using this software and we can see the pattern here is to ignore all requests. Was once great software and could rarely be beaten now it is outdating itself, with win 10 and 11 as primary OS here they are all exhibiting similar bugs, including the non ability to handle exceptions and forcing the user to close and loose all work, with something silly as a node colour change. so far something as simple as a license PR is ignored so I would not hold your breath on that. best to fork it and if you are taking over future dev, please ping me so I can follow your development and updates.
